packages/twenty-front/src/modules/action-menu/actions/record-actions/utils/isBulkRecordsManualTrigger.ts

```diff
@@ -0,0 +1,8 @@
+import { type WorkflowTrigger } from '@/workflow/types/Workflow';
+
+export const isBulkRecordsManualTrigger = (trigger: WorkflowTrigger) => {
```

Comment:
nitpick: we could narrow the type like so:

```suggestion
export const isBulkRecordsManualTrigger = (trigger: WorkflowTrigger): trigger is WorkflowTrigger & { settings: { availability: { type: 'BULK_RECORDS' } } } => {

```

Ideally, we would use a named type like `WorkflowBulkRecordsManualTrigger`